frontend/src/content/news.json

```diff
@@ -1,15 +1,32 @@
 {
+  "2022": [
+    {
+      "date": "2022-02-16",
+      "title": "Metabolic Atlas 2.3 adds info about the project",
+      "text": "<b>Metabolic Atlas 2.3</b> is out. In this incremental minor release, we have improved both the layout and the content on the <i>About</i> page, which now spans multiple sub-pages with a design similar to the existing <i>Documentation</i>. Moreover, we have resolved bugs in the <i>GEM Browser</i> and the <i>Map Viewer</i>, and improved documentation-related content. In addition, the process of deploying the software stack has seen some much-wanted attention to improve the robustness, thus paving the way for more regular releases. More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/2.3' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
```

Comment:
This link leads to a page where it is about 2.2. I'm not sure how zenodo works, would this auto update to the 2.3 stuff after release?

Reply:
That's right. Zenodo provides 2 types of DOIs (and urls): one for each individual release, and another for the _latest_ (a bit like the HEAD point with git). Since the release is not yet made, I can't include use the per-release DOI, so I'm using the _latest_ one. I have updated though the link to the 2.2 release below. @nanjiangshu had reported in the past that all Zenodo links were pointing to the latest, so in an earlier version of this file I updated all Zenodo links so that they point to the correct release.

Reply:
Thanks! Now that you explain it I'm pretty sure I have asked about this before too. Sorry about that. 😅